docs/exceptions/exceptions-logs.md

```diff
@@ -52,4 +56,13 @@ The table below indicates which attributes should be added to the
 Same as [Trace Semantic Conventions for Exceptions - Stacktrace
 Representation](exceptions-spans.md#stacktrace-representation).
 
+## Guidelines for Recording Exceptions in Spans, Logs, and Events
```

Comment:
Is this part intended to be Experimental or Stable for now?

Reply:
Recording of exceptions in events is experimental as it's being added only now.

Reply:
here's an example: https://github.com/open-telemetry/semantic-conventions/pull/989/files#diff-e1471b4097d9b5e1547732314a879b440712be6e2aa5d8b61013161018cafd90R7